community/caddy/APKBUILD

```diff
@@ -53,15 +54,13 @@ package() {
 		"$pkgdir"/etc/caddy \
 		"$pkgdir"/var/www
 
-	install -Dm755 "$srcdir"/$pkgname.initd \
-		"$pkgdir"/etc/init.d/$pkgname
```

Comment:
Why are installation of init.d removed here?

Reply:
I guess because they provide own https://github.com/mholt/caddy/tree/master/dist/init

Reply:
I don't see any openrc related files in that location.